contracts/interfaces/root/staking/ICustomSupernetManager.sol

```diff
@@ -21,6 +21,7 @@ interface ICustomSupernetManager {
     event RemovedFromWhitelist(address indexed validator);
     event ValidatorRegistered(address indexed validator, uint256[4] blsKey);
     event ValidatorDeactivated(address indexed validator);
+    event AccountPremined(address indexed account, uint256 amount);
```

Comment:
Update the event name.

Reply:
Fixed and added amount also as an indexed parameter https://github.com/0xPolygon/core-contracts/pull/411/commits/2d1dcc0317ce42c7ded9c8535eccec1633b4bed2